dask/dataframe/io/io.py

```diff
@@ -240,7 +240,7 @@ def from_bcolz(x, chunksize=None, categorize=True, index=None, lock=lock,
                     np.issubdtype(x.dtype[name], np.unicode_) or
                     np.issubdtype(x.dtype[name], np.object_)):
                 a = da.from_array(x[name], chunks=(chunksize * len(x.names),))
-                categories[name] = da.unique(a)
+                categories[name] = np.unique(a)
```

Comment:
Feel a little silly change this to use NumPy in this way. However this is consistent with the results it was getting before making `unique` lazy. Maybe someone more familiar with this code may want to take a look. According to GitHub, maybe @mrocklin?

Reply:
I might use `da.unique(a).compute()` instead.  These differ in that `da.unique` will call `np.unique` on each chunk before bringing them all into memory while `np.unique` will bring in the full original numpy array and then call `np.unique`.  If the original array is large then this will be inefficient.

Reply:
Yeah, was thinking about this as well. This should recover the original behavior nicely.